overlord/state/task.go

```diff
@@ -124,6 +124,11 @@ func (t *Task) SetStatus(s Status) {
 	t.status = s
 }
 
+// State returns the pointer to the system State
```

Comment:
Good catch and sorry that I failed to spot it in the previous commit. Fixed now :)